recipe/meta.yaml

```diff
@@ -1,13 +1,16 @@
 {% set name = "openmm-torch" %}
-{% set version = "0.6" %}
+{% set version = "0.7" %}
+
+# see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  git_url: https://github.com/openmm/{{ name }}.git
-  git_rev: v{{ version }}
+  url: https://github.com/openmm/openmm-torch/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 06a0f567c1b82746841424d63627e311d93cba4ab0e121f7d9d64701e753467a
 
 build:
   number: 0
```

Comment:
what happens if we remove the `rpaths` from below? did we try this in this PR? Do we also need the `missing_dso_whitelist`?